src/styles/components/_code-pattern.scss

```diff
@@ -0,0 +1,37 @@
+@import '../settings/colors';
+
+// =============================================================================
+// CODE PATTERN component
+//
+// A component that displays the code and a demo side by side.
+//
+// =============================================================================
+
+.code-pattern {
+  width: 100%;
+
+  .code-pattern__content {
+    display: grid;
+    gap: 1rem; // sass-lint:disable-line no-misspelled-properties
+    grid-template-columns: 50% auto;
+    justify-items: stretch;
+    height: 300px;
+    margin-bottom: 1rem;
+    overflow: scroll;
+  }
+
+  .code-pattern__assets,
+  .code-pattern__demo {
+    border: 1px solid $BORDER_COLOR;
+
+    iframe,
+    web-tabs {
+      height: 100%;
+      width: 100%;
+    }
+  }
+
+  .w-button {
+    text-transform: none;
+  }
```

Comment:
I think we would leave it with the uppercase styling that other buttons have on the site. I think that may be different from Jiwoong's mock, but that might be because he forgot. I think newer Material buttons don't do the uppercase thing, but the old ones did and we should probably be consistent with the rest of our UI.